router/router.gno

```diff
@@ -36,9 +38,9 @@ func SwapRoute(
 	}
 
 	switch swapType {
-	case ExactIn:
+	case "EXACT_IN":
 		amountSpecified = amountSpecified
-	case ExactOut:
+	case "EXACT_OUT":
```

Comment:
This change was made following the decision from yesterday's [disscussion](<https://onbloc.slack.com/archives/C0391ED6TPZ/p1710323498375949?thread_ts=1710159214.659069&cid=C0391ED6TPZ>) to have the parameters of the public function without a receiver be primitive types, right?